pr2_lfd_vision/IndexedBoundingBox indexed_bb
---
bool success
